packages/search/package.json

```diff
@@ -64,7 +64,7 @@
 	},
 	"scripts": {
 		"clean": "tsc --build ./tsconfig.json ./tsconfig-cjs.json --clean && rm -rf dist",
-		"build": "tsc --build ./tsconfig.json ./tsconfig-cjs.json && copy-assets",
+		"build": "tsc --build ./tsconfig.json ./tsconfig-cjs.json",
```

Comment:
Could you help me understand the reasoning of this change?

Reply:
Ah sorry this was preventing me from running the build so changed it temprorarily. Didnt mean to commit! Nice catch!